config/scopes.yml

```diff
@@ -7,6 +7,8 @@ optional_scopes:
   - email
   - reporting_access
   - transition_checker
+  - level0
+  - level1
```

Comment:
These should also be added to the `hidden_scopes`, as they're not really permissions the user is granting, they're just a mechanism to direct the auth process.

Reply:
ah ha, great will mvoe those over.
Have we got a link anywhere to the differences between the two (in terms of how we use them and what the user sees?)

I wasn't sure?

Reply:
The `hidden_scopes` don't show up on the consent screen, and if all the requested scopes are `hidden_scopes` then we don't show the consent screen at all:

https://github.com/alphagov/govuk-account-manager-prototype/blob/7858b55afcddc2f855d909abd051838d95f21f13/config/initializers/doorkeeper.rb#L453-L459

Reply:
They need to be in both `hidden_scopes` and `optional_scopes`